Tests/scripts/infrastructure_tests/test_collect_tests.py

```diff
@@ -157,7 +159,7 @@ def _test(monkeypatch, case_mocker: CollectTestsMocker, collector_class: Callabl
     if expected_tests is not None:
         assert collected.tests == set(expected_tests)
 
-    assert collected.packs_to_install == set(expected_packs or ()) | set(ALWAYS_INSTALLED_PACKS)
```

Comment:
why?

Reply:
Because when using `UploadAllPacks` i only want to upload and not install all packs, so this will always fail the upload all test.